test/aave/Close.test.ts

```diff
@@ -1,5 +1,6 @@
 import { JsonRpcProvider } from '@ethersproject/providers'
 import { TYPICAL_PRECISION } from '@oasisdex/oasis-actions'
+import { PositionType } from '@oasisdex/oasis-actions/lib/src/strategies/types'
```

Comment:
we should export that just from `@oasisdex/oasis-actions/strategies/types` or something like that and no import form the `src` folder.

Reply:
Funnily enough that import needs updating anyway.

Trying to update all imports to /oasis-actions/src -> so that tests and rest of repo are not dependent on the library being built first.